pages/pil/index.js

```diff
@@ -5,20 +5,22 @@ module.exports = settings => {
   const app = Router({ mergeParams: true });
 
   app.param('pilId', (req, res, next, pilId) => {
-    if (pilId === 'create') {
-      return next('route');
-    }
```

Comment:
How come this is no longer needed?

Reply:
I'm guessing because `/create is now mounted on `/pil` and not on `/pil/:id`.

Reply:
yup